cmd/zdb/zdb.c

```diff
@@ -6812,12 +6812,15 @@ dump_block_stats(spa_t *spa)
 
 	if (dump_opt['b'] >= 2) {
 		int l, t, level;
+			char csize[32], lsize[32], psize[32], asize[32];
```

Comment:
One tab to much ... next line also.

Reply:
I neglected to reply or say thank you, but I corrected this a while ago, I think.